src/__demo__/OpenFileDialog.stories.js

```diff
@@ -16,36 +36,48 @@ const user = {
 
 const onFileSelect = (id) => alert(`Opening ${id}`)
 
-storiesOf('OpenFileDialog', module).add('List of visualizations', () => (
-    <Provider config={configMock}>
-        <OpenFileDialog
-            type="visualization"
-            onClose={Function.prototype}
-            onFileSelect={onFileSelect}
-            onNew={Function.prototype}
-            open={true}
-            currentUser={user}
-        />
-    </Provider>
-))
-storiesOf('OpenFileDialog', module).add('List of maps', () => (
-    <Provider config={configMock}>
-        <OpenFileDialog
-            type="map"
-            onClose={Function.prototype}
-            onFileSelect={onFileSelect}
-            onNew={Function.prototype}
-            open={true}
-            currentUser={user}
-        />
-    </Provider>
-))
+const DVfilterVisTypes = [
```

Comment:
I get the idea behind testing "real world" scenarios like this, however this needs to be maintained when the app updates to not become outdated. So from Analytics perspective, wouldn't it be better to just test generic cases (e.g. types + divider, types + disabled etc) and then let DV and ER do their own app specific testing in Cypress to make sure the output matches the spec?

Reply:
Hm realising now that these aren't tests, they're stories. Still, the same principle applies. Better to have Analytics show off the available input in a generic fashion rather than tying the stories to apps?

Reply:
This stories rely on the api to show the list of AOs.
So the AO type is passed and at that point it makes sense to have the matching vis types list you would expect to see based on the AO type.
To make the whole thing generic we would have to mock everything and not use the api.
Perhaps for another PR 😁